narwhal/types/src/primary.rs

```diff
@@ -699,7 +699,7 @@ impl Certificate {
     Ord,
     Arbitrary,
 )]
-pub struct CertificateDigest([u8; crypto::DIGEST_LENGTH]);
+pub struct CertificateDigest(pub(crate) [u8; crypto::DIGEST_LENGTH]);
```

Comment:
Let's just add some accessor method to CertificateDigest,
Something like `into_bytes(self) -> [u8; crypto::DIGEST_LENGTH]` if we need to extract value, or `as_bytes(&self) -> &[u8; crypto::DIGEST_LENGTH]` if we need a reference